sql/init-index-db.sql

```diff
@@ -2,12 +2,14 @@ drop table if exists crates cascade;
 create table crates
 (
     id            integer primary key generated always as identity,
-    name          text not null,
-    registry      text not null default '',
+    name          text        not null,
+    registry      text        not null default '',
     description   text,
     documentation text,
     homepage      text,
     repository    text,
+    created_at    timestamptz,
```

Comment:
Nullable because it could be a dependency in crates.io rather than an actual crate.